core/client/ui/toolboxes/entity-template-list.js

```diff
@@ -1,6 +1,14 @@
 import { generateEntityThumbnail } from '../../helpers';
 
+const prefabHideProperties = [
+  '_id',
+  'levelId',
+  'createdAt',
+  'createdBy',
+];
+
 const prefabEntities = () => Entities.find({ prefab: true }).fetch();
+const getAllInstances = (key) => Entities.find({ 'gameObject.sprite.key': key }).fetch();
```

Comment:
I think we'll have to add an entityId/prefabId on the instances later because relying on this attribute doesn't look good, I should have thought about it at the beginning 😕